### ISSUE TITLE
Processing 4 reaches "auto naming limit", suggests going for a walk

### ISSUE THREAD

**Issue by rric** [benfry/processing4#582]
A student of mine struggeled with Processing 4 today, and ran into the following behaviour. I was able to reproduce it:

## Steps to Reproduce

* start Processing, close it, re-start it again; repeat several times
 
or, 

* start Processing, and create a number[^1] of new sketches 

[^1]: this number might be 26, as the last sketch I was able to create ended with `z`.

Then, at some point, the following dialog pops up:

![Time for a Break: You've reached the limit for auto naming of new sketches for the day. How about going for a walk instead?](https://user-images.githubusercontent.com/922274/197599511-9bee2c5e-3408-4a49-ac07-38b4b3c6ba9a.png)

When trying to start Processing again, one gets this:

![Sunshine: No really, time for some fresh air for you.](https://user-images.githubusercontent.com/922274/197598435-753d5f76-9ed0-42ae-a6a0-0f49d9f6be61.png)

That's it. One can **no longer start** Processing for an unknown period of time.

What is that, some kind of an Easter egg?! I find this rather inappropriate. Today, it only led to even more frustration for my students who are just starting to learn Python and Processing.

## Expected Behavior

Please don't tell me Processing cannot "auto name" more than 26 sketches per day. Keep the funny messages, if you like, but allow Processing to start again. Thanks.

##  Environment

* Processing version: 4.0.1
* Operating System and OS version: Windows 10

**Comment by SableRaf** [comment]
Hello @rric

Here's how you may fix this on Windows 10:

1. Go to `C:\Users\[User name]\AppData\Local\Temp\processing`
2. Move the anonymous sketches (named like `sketch_000000a` to `sketch_000000z`) to another folder.
3. Start Processing again. It should open normally.

You may also manually create an empty `.pde` file outside of the Processing IDE and double click it.

The best way to avoid reaching the maximum auto-naming limit is to save your sketches.

I hope this will help your students and hopefully anyone else who may run into this issue in the future.

---

For anyone interested the check happens from line 70 in [SketchName.Java](https://github.com/processing/processing4/blob/9b46aab2ef4e5a1a88da4d26df5f30e2db217398/app/src/processing/app/SketchName.java)

```Java
    if (index == 26) {
      // In 0159, avoid running past z by sending people outdoors.
      if (!breakTime) {
        Messages.showWarning("Time for a Break",
                "You've reached the limit for auto naming of new sketches\n" +
                        "for the day. How about going for a walk instead?", null);
        breakTime = true;
      } else {
        Messages.showWarning("Sunshine",
                "No really, time for some fresh air for you.", null);
      }
      return null;
    }
  ```

**Comment by rric** [comment]
I was able to reproduce a similar behaviour in Processing 3.5.4, i.e. this issue seems to be older ... Surprising, then, that I was not able to find any reference beforehand. 

@SableRaf, thanks for explaining and pointing out the workaround.

**Comment by benfry** [comment]
Fixed for 4.0.2; this was just a regression introduced by the changes to temp folder handling to resolve https://github.com/processing/processing4/issues/529.
